marlowe-integration/src/Test/Integration/Marlowe/Local.hs

```diff
@@ -285,6 +285,7 @@ withLocalMarloweRuntime' MarloweRuntimeOptions{..} test = withRunInIO \runInIO -
     webPort <- liftIO $ randomRIO (4000, 5000)
     manager <- liftIO $ newManager defaultManagerSettings
 
+    let submitConfirmationBlocks = 2
```

Comment:
If we still have troubles with the integration tests on the fast network, we can increase this further.

Reply:
Yeah, this is our knob. This is an "optimistic" initial value